sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -374,15 +374,15 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
       sql(s"CREATE TABLE test_partition (a STRING) PARTITIONED BY (b BIGINT, c STRING)")
       sql(s"CREATE TABLE ptest (a STRING, b BIGINT, c STRING)")
 
-      val analyzedPlan = sql(
+      val optimizedPlan = sql(
```

Comment:
Why does this use the optimized plan now? So that `checkAnalysis` will run?

Reply:
The `Cast` operator will be eliminated by optimization rule `ConstantFolding`.

Reply:
Why not update the test case with the Cast operator?